lib/seattleflu/db/cli/command/etl/__init__.py

```diff
@@ -140,4 +140,76 @@ def upsert_encounter(db: DatabaseSession,
     return encounter
 
 
+def update_kit_samples(db: DatabaseSession, kit: dict):
+    """
+    After upserting kit, update the samples linked to the kit.
+    """
+    if kit.rdt_sample_id:
+
+        rdt_sample = find_sample_by_id(db, kit.rdt_sample_id)
+        
+        if rdt_sample:
+            update_sample(db,
+                sample         = rdt_sample,
+                encounter_id   = kit.encounter_id)
+
+    if kit.utm_sample_id:
+
+        utm_sample = find_sample_by_id(db, kit.utm_sample_id)
+        
+        if utm_sample:
+            
+            update_sample(db,
+                sample         = utm_sample,
+                encounter_id   = kit.encounter_id)
+
+
```

Comment:
Did you mean to add `update_kit_samples` here only to delete it (specifically move it to `etl/kit` in commit d28106?

Reply:
Yea, after restructuring the `update_kit_samples` is now only used by the kit etl so it no longer needs to be in the shared module